blazor/chart/getting-started-wasm.md

```diff
@@ -19,17 +19,23 @@ This section briefly explains about how to include [Blazor Charts](https://www.s
 
 ## Create a new Blazor App in Visual Studio
 
-You can create **Blazor WebAssembly App** using Visual Studio in one of the following ways,
+You can create a **Blazor WebAssembly App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio).
 
-* [Create a Project using Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?pivots=windows&view=aspnetcore-7.0)
+## Install Syncfusion Blazor Charts and Theme NuGet in the App
```

Comment:
Did checked without Theme NuGet?

Reply:
Yes, checked this component without Theme NuGet also Theme NuGet was not needed . So Theme NuGet was removed from this PR.